pkg/mindpak/testdata/t2/manifest.json

```diff
@@ -1,16 +1,14 @@
 {
```

Comment:
I repurpose this test data for my own tests. The changes did not affect any existing tests.